pkg/openapi-gen/openapi-gen.go

```diff
@@ -636,8 +649,6 @@ func newPathItem(dirEntry *yang.Entry, itemPath string, parentPath string, pathT
 	if pathType == pathTypeContainer {
 		getOp.OperationID = fmt.Sprintf("get%s", toUnderScore(itemPath))
 	}
-	getOp.Tags = []string{toUnderScore(parentPath)}
```

Comment:
I think this was good to have, as it caused the Endpoints to be grouped per parent, but we can revisit once the future plans are clearer